<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsGui</class>
 <widget class="QWidget" name="StatisticsGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StatisticsGui</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="yearlabel">
      <property name="text">
       <string>year:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="yearSelect"/>
    </item>
    <item>
     <widget class="QLabel" name="semesterlabel">
      <property name="text">
       <string>semester</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="semesterSelect"/>
    </item>
    <item>
     <widget class="QLabel" name="classLabel">
      <property name="text">
       <string>class </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="classSelect"/>
    </item>
    <item>
     <widget class="QLabel" name="sectionlabel">
      <property name="text">
       <string>section</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sectionSelect"/>
    </item>
    <item>
     <widget class="QLabel" name="labnolabel">
      <property name="text">
       <string>lab number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="labNoSelect"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="meanlabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>mean</string>
   </property>
  </widget>
  <widget class="QLabel" name="meandisplay">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="medianlabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>median</string>
   </property>
  </widget>
  <widget class="QLabel" name="mindisplay">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="calcbutton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxdisplay">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mediandisplay">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>350</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>done</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxlabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>max</string>
   </property>
  </widget>
  <widget class="QLabel" name="minlabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
